frontend/src/routes/manage/Video/Shared.tsx

```diff
@@ -31,6 +31,7 @@ export const makeManageVideoRoute = (
     page: ManageVideoSubPageType,
     path: string,
     render: (event: AuthorizedEvent, data: QueryResponse) => JSX.Element,
+    options?: { onAccessRoute?: boolean },
```

Comment:
I would probably call this just sth like `fetchWorkflowState` or sth more direct?